app/src/components/main/breadcrumbs/breadcrumb/breadcrumb.scss

```diff
@@ -5,6 +5,7 @@
   padding-right: 5px;
   font-size: 12px;
   color: $COLOR--charcoal-grey;
+  min-width: 15px;
```

Comment:
Do these changes do what the task requires?

Reply:
they don't but the rule 'word-break' does. It was rebased from another commit in v5 and merged here so only one line is remaining